scripts/tract_math

```diff
@@ -35,7 +37,29 @@ def main():
     from tract_querier.tract_math import TractMathWrongArgumentsError
 
     usage = r"""
-    usage: %(prog)s <tract1.vtk> ... <tractN.vtk> operation <operation parameter1> ... <operation parameterN> <output_tract.vtk>
+    usage: %(prog)s <tract1.vtk> ... <tractN.vtk> operation <operation parameter1> ... <operation parameterN> <output_tract.vtk> \
+                      [ optional flags ]
+
+    optional flags:  Always start with "--" and are considered as boolean flags
+
+    The --useFileNamesAsIndex col_name1:path_dir_index col_name2:path_dir_index2 .. col_nameN:path_dir_indexN
+        Example directory layout:
```

Comment:
maybe we could switch the naming code of the parameter to:
```
--use_file_name_as_index
```
to be more consistent with the names of all other operations?